docs/conf.py

```diff
@@ -1,9 +1,9 @@
 import os
 on_rtd = os.environ.get('READTHEDOCS') == 'True'
 if on_rtd:
-    html_theme = 'default'
-else:
     html_theme = 'sphinx_rtd_theme'
+else:
```

Comment:
Not 100% sure of the original intent here, but `sphinx_rtd_theme` seems more consistent.